[DO NOT MERGE] DOCS-12770 lifecycle states (#166)
mulesoft/docs-exchange: modules/ROOT/pages/index.adoc

:imagesdir: ../assets/images/
:page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc

Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, templates, examples, custom assets, and integration assets such as connectors. You can catalog (publish), share, discover, learn about, and reuse assets within your organization to facilitate collaboration, boost productivity, and promote standards.

image:exchange-landing-page.png[Exchange landing page]

Use Exchange to discover, share, and incorporate assets and resources into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.

> Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), and Runtime Manager (APIs), and you can publish APIs from Design Center to Exchange.

Reviewer: ```suggestion
Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), and Runtime Manager (Applications), and you can publish APIs from Design Center to Exchange.
```

:imagesdir: ../assets/images/
:page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc

Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, templates, examples, custom assets, and integration assets such as connectors. You can catalog (publish), share, discover, learn about, and reuse assets within your organization to facilitate collaboration, boost productivity, and promote standards.

image:ex2-old-search-landing.png[Exchange landing page]

Use Exchange to discover, share, and incorporate assets and resources into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.

> Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), and Runtime Manager (applications), and you can publish APIs from Design Center to Exchange.

Reviewer 2: I wouldn't enumerate here since it's not an exhaustive list, assets are also accessible in other ways such as Graph Service, Maven Facade, Design Center API Designer

> Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), and Runtime Manager (applications), and you can publish APIs from Design Center to Exchange.

Reviewer: ```suggestion
Exchange assets are accessible from many places like Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), or Runtime Manager (applications). APIs can be published from Design Center to Exchange.
```
@sancheztavo what so you think?

:imagesdir: ../assets/images/
:page-aliases: general:getting-started:anypoint-exchange.adoc, ex-ref-raml.adoc

Anypoint Exchange is a curated catalog of reusable assets such as APIs, policies, API spec fragments, API groups, templates, examples, custom assets, and integration assets such as connectors. You can catalog (publish), share, discover, learn about, and reuse assets within your organization to facilitate collaboration, boost productivity, and promote standards.

image:ex2-old-search-landing.png[Exchange landing page]

Use Exchange to discover, share, and incorporate assets and resources into your applications. Exchange helps you create API developer portals, view and test APIs, simulate sending data to APIs by using the mocking service, create assets, and use API Notebooks to describe and test API functions.

Exchange assets are accessible in Anypoint Studio (connectors, templates, and examples), API Manager (APIs, API groups, and policies), and Runtime Manager (applications), and you can publish APIs from Design Center to Exchange.

> Assets under the *Provided by MuleSoft* organization are created by MuleSoft and its partners, and are certified by Mulesoft. These assets are publicly available to all customers as well. However, all assets that are created by an organization and published into the organization's instance of Exchange, are private and not visible outside that organization. Customers are responsible for any private assets they publish within their Exchange instance.

Reviewer 3: ```suggestion
Assets in the *Provided by MuleSoft* organization are created by MuleSoft and its partners and certified by Mulesoft. These assets are publicly available to all customers. However, assets that are created by an organization other than *Provided by MuleSoft* and published in that organization's instance of Exchange are private and not visible outside that organization. Customers are responsible for any private assets they publish within their Exchange instance.
```
Should we explain what it means to be "responsible for"?

== Get Started

> 

Reviewer 3: It is best to provide some narrative context between the title and Step 1 of a task. Be sure to add value to the context beyond restating the title.

> . From https://www.anypoint.mulesoft.com/exchange/[Exchange], search for assets. If you know the name of the asset, enter the name in the *Search* field. 

Reviewer 3: ```suggestion
. From https://www.anypoint.mulesoft.com/exchange/[Exchange], either search for assets or enter the name of a known asset in the *Search* field. 
```

Reviewer 3: It is not always possible, but it is best to limit steps in a task to one sentence.

> . Select an asset from the search results view.

Reviewer 3: If you search for a known asset, is only that one asset shown? Does this step apply to both circumstances provided in Step 1?

. Review the description of the asset.

> To create assets for APIs, connectors, templates, examples, API Notebooks, and custom assets, log into Exchange. See xref:workflow-sharing-assets.adoc[Workflow: Publish APIs] for more information.

Reviewer 3: Why are we including creation info in what has so far been a search task?
Maybe this Get Started <with what?> topic should begin by saying that you can get started either by creating a new asset for using an existing one, and then go on from there?

> == Try APIs

Reviewer 3: I don't know what this title means. "Test" might be a better word, or "Find the Right API for Your Environment"?
Also could use a transition from title to Step 1.

> . Click the *All Types* filter, check the REST APIs option, and click *Apply*.

Reviewer 3: ```suggestion
. Click the *All Types* filter, select the REST APIs option, and click *Apply*.
```
Per the style ref, use "select" and "unselect" for check boxes.

. Click the asset and notice how the API functions appear in the left

> navigation bar. Exchange lets you view the functions in an API.

Reviewer 3: Does the second sentence add anything to the first?

. Click a function and more information appears. At the right side of the screen, you can test the API by entering sample values to send to the API and get responses from the API.

> == Need More Information on an Asset?

Reviewer 3: Our style is to not use sentences, especially questions, as titles.

Click the Helpful Links item in the left navigation area for links to documentation.

== Language Support

Using multiple languages with Unicode is supported fully throughout Exchange and in all metadata fields, including asset names, descriptions, markdown pages, categories, tags, and reviews.

== Versions

> This page applies to the current version, Exchange 2.

Reviewer 3: Please reconcile this with "the API Exchange version 2" (or 1) in other parts of the narrative
